### DISCUSSION TITLE
Check boxes in Rich Text Plus Editor

### DISCUSSION THREAD

**Discussion by fury182** [standardnotes/forum#2786] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**

Yes

**Is your feature request related to a problem? Please describe.**

No.

**Describe the solution you'd like**

I would like to have the option to include check boxes in the Rich Text Plus Editor. This is essential to the OneNote experience for example (where they are called "To Do", but it's basically just a check box that can be checked and unchecked). And similarly in Standard Notes it would help users create to do lists with more freedom (multiple lists in the same note, indented as the user desires etc.)

**Describe alternatives you've considered**

Alternatives would be rich text editor extensions that have it, but then you'd depend on the security and longevity guarantees of that particular developer. And this would be a small feature to include in Standard Notes.

**Additional context**

n/a

**Comment by stefan-yas** [comment]
Depending on your plan, you may use the Task editor, if you need it for to-do lists. If that's not your use-case, then supporting checkboxes in the Plus editor would need to come from [Summernote](https://github.com/summernote/summernote), where it is derived from.

**Comment by fury182** [comment]
@stefan-yas I have Task Editor but the problem with it is that it is too simple. We can't have multiple task lists with a heading inside the same note for example, so only the most basic needs for to do/tasks are addressed by it.

**Comment by fury182** [comment]
I will also open an issue for Summernote

**Comment by fury182** [comment]
I opened an issue here: https://github.com/summernote/summernote/issues/4245 and someone pasted in the comments a one-page-of-code Summernote plugin that adds the checkboxes functionality. Do you think something like it could be included in Standard Notes? Or must it be in the core Summernote? Thanks

**Comment by stefan-yas** [comment]
Thanks for that, @fury182! I'll relay to my team to see how complicated it is to have this functionality inside the Plus editor.

**Comment by DennisSuitters** [comment]
I have answered the issue on Summernote as per the opened issue above, with a solution via plugin. But there has been no further activity. I am commenting here, so the answer helps other users of your product, or your developers, if they wish to use the solution.
